tests/attention/test_asymmetric_kv_decode.py

```
@@ -0,0 +1,382 @@
+    kv_len,
+    page_size,
+    num_kv_heads,
+    num_qo_heads,
+    head_dim,
+    kv_layout,
+    k_dtype,
+    v_dtype,
+):
+    torch.manual_seed(42)
+    q = torch.randn(batch_size, num_qo_heads, head_dim, device="cuda:0", dtype=k_dtype)
+    k_cache, k_fp32, num_pages_per_seq, total_num_pages = _build_paged_cache(
```

Ruff (0.15.20)

[warning] Unpacked variable `k_fp32` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)